arch/HPPA/HPPADisassembler.c

```diff
@@ -1167,7 +1177,7 @@ static void fillIdxmemMods(uint32_t insn, hppa_ext* hppa_ext, cs_mode mode, uint
     uint32_t cmplt = (GET_BIT(insn, 18) << 1) | GET_BIT(insn, 26);
     uint32_t cc = GET_FIELD(insn, 20, 21);
     uint32_t ext = GET_FIELD(insn, 22, 25);
-    if (cmplt & 1 == 1) {
+    if ((cmplt & 1) == 1) {
```

Comment:
Please replace the `1` in `cmplt & 1` with some macro named after the flag checked here.